src/app/overview/widgets/coldstake/coldstake.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-coldstake',
+  templateUrl: './coldstake.component.html',
+  styleUrls: ['./coldstake.component.scss']
+})
+export class ColdstakeComponent implements OnInit {
+
+  constructor() { }
+
```

Comment:
Can remove the constructor and the ngOnInit

Reply:
Keeping it in for now, won't be empty for long.